service/application/src/main/java/com/solace/maas/ep/event/management/agent/processor/CommandLogStreamingProcessor.java

```diff
@@ -0,0 +1,208 @@
+package com.solace.maas.ep.event.management.agent.processor;
+
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.solace.maas.ep.common.messages.CommandLogMessage;
+import com.solace.maas.ep.event.management.agent.config.eventPortal.EventPortalProperties;
+import com.solace.maas.ep.event.management.agent.plugin.command.model.Command;
+import com.solace.maas.ep.event.management.agent.plugin.command.model.CommandRequest;
+import com.solace.maas.ep.event.management.agent.plugin.command.model.JobStatus;
+import com.solace.maas.ep.event.management.agent.plugin.terraform.manager.TerraformLogProcessingService;
+import com.solace.maas.ep.event.management.agent.publisher.CommandLogsPublisher;
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.MDC;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import static com.solace.maas.ep.event.management.agent.constants.Command.COMMAND_CORRELATION_ID;
+import static com.solace.maas.ep.event.management.agent.plugin.constants.RouteConstants.ACTOR_ID;
+import static com.solace.maas.ep.event.management.agent.plugin.constants.RouteConstants.TRACE_ID;
+
+@Component
+@Slf4j
+public class CommandLogStreamingProcessor {
+
+    public static final String ANY = "*";
+    private final CommandLogsPublisher commandLogsPublisher;
+    private final EventPortalProperties eventPortalProperties;
+    private final ObjectMapper objectMapper;
+
+    public CommandLogStreamingProcessor(CommandLogsPublisher commandLogsPublisher,
+                                        EventPortalProperties eventPortalProperties,
+                                        ObjectMapper objectMapper) {
+        this.commandLogsPublisher = commandLogsPublisher;
+        this.eventPortalProperties = eventPortalProperties;
+        this.objectMapper = objectMapper;
+    }
+
+
+    public void deleteExecutionLogFiles(List<Path> listOfExecutionLogFiles) {
+        listOfExecutionLogFiles.forEach(path -> {
+            try {
+                if (Files.exists(path)) {
+                    Files.delete(path);
+                }
+            } catch (IOException e) {
```

Comment:
I'm wondering if we just log a warning in this case instead of throwing an exception since we want to loop through all of the files and not interrupt the flow if an error occurs. Or, wait to throw the exception until we've gone through the entire list.

Reply:
Updated the code